finality-aleph/src/network/manager/service.rs

```diff
@@ -558,14 +558,14 @@ pub struct IO<
     D: Data,
     M: Data,
     A: AddressingInformation + TryFrom<Vec<M>> + Into<Vec<M>>,
-    VN: ValidatorNetwork<A::PeerId, A, DataInSession<D>>,
+    CN: CliqueNetwork<A::PeerId, A, DataInSession<D>>,
     GN: GossipNetwork<VersionedAuthentication<M, A>>,
 > where
     A::PeerId: PublicKey,
 {
     commands_from_user: mpsc::UnboundedReceiver<SessionCommand<D>>,
     messages_from_user: mpsc::UnboundedReceiver<(D, SessionId, Recipient)>,
-    validator_network: VN,
+    validator_network: CN,
```

Comment:
Have you left it named `validator_network` here and in other places on purpose?

Reply:
Yes, in here it is actually the validator network, because it is for the validators specifically, not just an abstract clique network.

Reply:
Got it